### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jt-nti**
Combine FAQ with readme and ensure information is up-to-date

Signed-off-by: James Taylor <jamest@uk.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,75 @@
-# Hyperledger Fabric Shim for Java chaincode
```
Comment (heatherlp) [inline README.md]: Good point on removing "shim" - Anthony will appreciate :)

File: README.md [hunk 2]
```diff
@@ -1,30 +1,75 @@
...
-## Folder structure
+## Getting Started
+
+Application developers interested in developing Java smart contracts for Hyperledger Fabric should read the [JavaDoc](https://fabric-chaincode-java.github.io/) which inludes download information, and links to documentation and samples.
+
+## Project structure
+
+### fabric-chaincode-protos
+
+Contains the protobuf definition files used by Java shim to communicate with Fabric peer.
```
Comment (heatherlp) [inline README.md]: "with a fabric peer" "with THE fabric peer"? :)

Comment (heatherlp) [reply README.md]: Line 25 too

Comment (jt-nti) [reply README.md]: Gone with "Fabric peers" in the expectation that there'll be lots of them :)

**Pull request merged**